values/k8s/k8s-raw.gotmpl

```diff
@@ -13,7 +13,13 @@ resources:
       name: {{ $ns.name }}
       labels:
         name: {{ $ns.name }}
-        {{ if not ($ns | get "disableIstioInjection" false) }}istio-injection: enabled{{ end }}
+        {{- if not ($ns | get "disableIstioInjection" false) }}
+        {{- if $v.apps.istio.defaultRevision }}
+        istio.io/rev: {{ $v.apps.istio.defaultRevision }}
```

Comment:
Is `istio.io/rev` equivalent to `istio-injection:` in this case?

Reply:
From intuition I would not consider it equivalent, but in order for the `MutatingWebhookConfiguration` from istiod to do its magic, it is required that **either** `istio-injection` or `istio.io/dev` is set.